src/controllers/handlers/resize-handler.ts

```diff
@@ -0,0 +1,62 @@
+import { HandlerContextWithPath } from '../../types'
+
+import createCommandTrigger from '../../adapters/commandLine'
+
+export async function resizeHandler(
```

Comment:
would make sense (it can be done in another PR) to handle the case of two simultaneous request asking to resize the same file?

Reply:
Hey, I like this idea, do you have any example/idea on how to address this? I'd like to include it in the next PR.

Reply:
Maybe you can have a map of <hash, processingPromise>, so any request can wait for the processing of a hash if already in progress? This is a bit hacky but it can be improved when we move to a workers system.

Reply:
Oh I got what you meant, I'll think about it, thank you!

Reply:
another option is to memoize the conversion function to make it idempotent

Reply:
I'll implement this in the next PR so I can unblock explorer team asap, thank you!